GraphsV2/GaugeChart/gaugeGraph.spec.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import { getDataAndConfig } from '../testHelper';
+
+import GaugeGraph from '.';
+
+const cheerio = require('cheerio');
+
+describe('GaugeGraph', () => {
+    let config;
+    let useEffect;
+
+    const mockUseEffect = () => {
```

Comment:
I suspect this will be used in many testcases let's create a library and the this mock function there.

Reply:
Hi Natalia, 
I don't see this as some generic function and also it is not being used in many places. If in future we will need it for multiple places, I will make it a generic function.

Reply:
@nxjiten The point is that I would like all the mock functions to be in one file and re-use. Please change and please consider clearAllMocks as well.